powermon/protocols/pi30.py

```diff
@@ -11,7 +12,7 @@
         "name": "F",
         "description": "Set Device Output Frequency",
         "help": " -- examples: F50 (set output frequency to 50Hz) or F60 (set output frequency to 60Hz)",
-        "response_type": ResponseType.ACK,
+        "response_type": ResultType.ACK,
```

Comment:
strictly this is a ack type response with a single result that is also an ack type

Reply:
There are two concepts here with the same name. I figured I would rename the higher level to a Result which contains different Responses. In this case the ResultType.ACK matches the ResponseType which is "ack". I will make a ResponseType enum and do a bulk replace to make it clear.

Reply:
My thinking was 'response' related to the response from the device, 'result' was the processed response, and '????' was each individual value/parameter